pages/checkoutPage/checkout.ts

```diff
@@ -0,0 +1,85 @@
+import { expect, Locator } from '@playwright/test';
+import { BasePage } from '../home/basePage';
+import { BillingStep } from './components/billing.step.component';
+import { PaymentStep } from './components/payment.step.component';
+import { envConfig } from '../../env.config';
+
+export type PaymentMethod =
+  | 'Bank Transfer'
+  | 'Credit Card'
+  | 'Cash on Delivery'
+  | 'Buy Now Pay Later'
+  | 'Gift Card';
+
+export class CheckoutPage extends BasePage {
+  readonly billingStep: BillingStep = new BillingStep(this.page);
+  readonly paymentStep: PaymentStep = new PaymentStep(this.page);
+
+  readonly checkoutTitle: Locator = this.page.getByTestId('product-title');
+  readonly productPrice: Locator = this.page.getByTestId('product-price');
+  readonly cardTotalPrice: Locator = this.page.getByTestId('cart-total');
+  readonly proceedToCheckoutButton: Locator = this.page.getByRole('button', {
+    name: 'Proceed to checkout',
+  });
+  readonly pageMessage: Locator = this.page.locator('p.ng-star-inserted');
+
+  async getCheckoutTitle(): Promise<string> {
+    const title = (await this.checkoutTitle.textContent())?.trim();
+    if (title) {
```

Comment:
I do not think we need to handle an error here. 

```suggestion
    return (await this.checkoutTitle.textContent())?.trim();
```

Should be enough

Reply:
The same below

Reply:
fixed